perf/lock.ts

```diff
@@ -18,34 +18,45 @@ export function benchmarks() {
   ];
 }
 
-export async function isHeld(name: string) {
-  return await navigator.locks.request(
+export async function isHeld(name: string): Promise<boolean> {
+  return (await navigator.locks.request(
     name,
     {ifAvailable: true},
     async lock => lock === null,
-  );
+  )) as unknown as boolean;
```

Comment:
Not sure why this is needed. Looks like the d.ts was meant to deal with this.

🤷

Reply:
The annotation is slightly off--it says that `request()` returns `undefined`, but really it returns the result of the inner callback.